src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -1203,8 +1214,19 @@ export default {
 			}
 
 			const basketItems = basketState.shop?.basket?.items?.values ?? [];
+			// insert some kind of tag to indicate that creditAllocationLoans should have a different message on them.
+
 			// eslint-disable-next-line no-underscore-dangle
 			this.basketLoans = basketItems.filter(item => item.__typename === 'LoanReservation');
+			this.basketLoans.forEach(item => {
+				if (item.id === this.leftoverCreditAllocationLoanId) {
+					const prevLCALoanId = this.cookieStore.get('lcaid');
+					if (this.leftoverCreditAllocationLoanId !== prevLCALoanId) {
+						return this.cookieStore.set('lcaid', this.leftoverCreditAllocationLoanId);
```

Comment:
This `return` just means "continue" in this context...so prob not needed.

Reply:
done